app/services/idv/steps/verify_step.rb

```diff
@@ -40,21 +40,17 @@ def enqueue_job
 
         idv_agent.proof_resolution(
           document_capture_session,
-          should_proof_state_id: should_use_aamva?(pii_from_doc),
+          should_proof_state_id: should_use_aamva?(pii),
           trace_id: amzn_trace_id,
         )
       end
 
-      def pii_from_doc
-        flow_session[:pii_from_doc]
-      end
-
       def idv_agent
-        @idv_agent ||= Idv::Agent.new(pii_from_doc)
+        @idv_agent ||= Idv::Agent.new(pii)
       end
 
       def invalid_state?
-        flow_session[:pii_from_doc].nil?
+        pii.nil?
```

Comment:
maybe do `pii.blank?` ? Because an empty hash is also invalid right?

```suggestion
        pii.blank?
```

Reply:
Done in e07b98fe2